tests/python/unittest/common.py

```diff
@@ -31,6 +31,7 @@
 from contextlib import contextmanager
 from nose.tools import make_decorator, assert_raises
 import tempfile
+import gc
```

Comment:
nit: we usually import generic modules before all the custom modules. So this is better to go in the second line.

Reply:
ok